src/app/resources/resources-add.component.html

```diff
@@ -78,7 +78,11 @@
         <input matInput i18n-placeholder [(ngModel)]="userDetail.name" placeholder="Added By" formControlName="addedBy" readonly>
       </mat-form-field>
       <mat-form-field>
-        <input matInput  i18n-placeholder placeholder="Open Which File" formControlName="openWhichFile">
+        <mat-select i18n-placeholder placeholder="Open Which File" formControlName="openWhichFile">
```

Comment:
Trying this out I think it might be to use [autocomplete](https://material.angular.io/components/autocomplete/overview) instead of a select.  Long select lists are a little difficult to navigate.

Reply:
autocomplete with filter looks good, ill try that out